app/models/micropost.rb

```diff
@@ -0,0 +1,20 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  has_one_attached :image
+
+  validates :user_id, presence: true
```

Comment:
validate này auto có khi e thêm belongs to ở dòng 2. nên xóa đi nhé

Reply:
rồi cô k xóa mà thả chuym thôi à :D

Reply:
ô rõ em xóa rồi ạ. để e check lại ạ :(